sass/components/_card.scss

```diff
@@ -1,13 +1,32 @@
 .card {
+    @include flex(column, left, left);
     position: relative;
-    max-width: 40rem;
+    max-width: 37rem;
     padding: 2rem;
-    margin-bottom: 35rem;
+    border-radius: 2.6rem;
+
+    z-index: 1;
+
+    &__disactivated {
```

Comment:
to nie jest element bloku, tylko modyfikator bloku,
`card--disactivated`

Reply:
zmieniono

Reply:
nie widze zmiany na modyfikator